Core/GDCore/Extensions/Builtin/VariablesExtension.cpp

```diff
@@ -207,10 +281,132 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsVariablesExtension(
                  _("Clear global variable"),
                  _("Remove all the children from the global variable."),
                  _("Clear children from global variable _PARAM0_"),
-                 _("Variables/Global variables/Structure"),
+                 _("Variables/Global variables/Collections"),
+                 "res/actions/var24.png",
+                 "res/actions/var.png")
+      .AddParameter("globalvar", _("Variable"))
+      .MarkAsAdvanced();
+
+  extension
+      .AddAction("SceneVariablePush",
+                 _("Push variable on a scene array"),
```

Comment:
Could we use `Append XXX to a YYY array` everywhere?
While it's named `push` in javascript, it seems the MDN docs never use "push" as a verb, always explaining things in terms of "append" or "add". 
Also make it easier for people to search the actions if they have the same words as the sentence.